Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/defaults.go

```diff
@@ -55,6 +55,14 @@ func addDefaultingFuncs() {
 			if obj.Protocol == "" {
 				obj.Protocol = ProtocolTCP
 			}
+
+			// Carry conversion to make port case valid
+			switch strings.ToUpper(string(obj.Protocol)) {
```

Comment:
Uh... probably here.

On Jul 6, 2015, at 6:41 PM, Jordan Liggitt <notifications@github.com> wrote:

In
Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/defaults.go
<https://github.com/openshift/origin/pull/3592#discussion_r33990835>:

> @@ -55,6 +55,14 @@ func addDefaultingFuncs() {
>  			if obj.Protocol == "" {
>  				obj.Protocol = ProtocolTCP
>  			}
> +
> +			// Carry conversion to make port case valid
> +			switch strings.ToUpper(string(obj.Protocol)) {

should I put these in the v1_to_api conversions or leave them here?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3592/files#r33990835>.